app/notifiers/notifier/base.rb

```diff
@@ -0,0 +1,79 @@
+# The base class of notifiers. This is meant to be called by the Notifications Framework
+#
+# @api notifications
+class Notifier::Base
+  class ActivityWrapper < SimpleDelegator
```

Comment:
hmm... seems no need to do this. And also it is more clear to put this wrapper in notifier because only the notifier will create the wrapper